src/app/ApplicationTemplate.Shared.Views/Styles/Controls/ExtendedSplashScreen.xaml

```diff
@@ -45,9 +48,9 @@
 				</ControlTemplate>
 			</Setter.Value>
 		</Setter>
-	</Style>
+	</not_win:Style>
 
-	<Style TargetType="splash:ExtendedSplashScreen"
-		   BasedOn="{StaticResource DefaultExtendedSplashScreenStyle}" />
+	<not_win:Style TargetType="splash:ExtendedSplashScreen"
```

Comment:
We don't have a style for Windows?

Reply:
@Marc-Antoine-Soucy

Reply:
If I remember correctly, WinUI doesnt have a native splashscreen, and making a fake one brought a lot of problems, so, we ended up not implementing it for WinUI, right, @takla21?

Reply:
If I recall, we discussed adding a property, but we didn't bother as it was impossible to test in the context of the sample (both WinUI & UWP). With that in mind, we prefered to focus on completing the .net6 migration

Reply:
The thing that is weird is that I can have the control be in my Shell in Windows but I can't have a style for it, otherwise it crash.

Reply:
Fixed, it was the content presenter.

Reply:
There's no native splashscreen in WinUI, but that just means nothing will show before the extended splashscreen.
The extended splash screen is going to be **the** splash screen on WinUI.
For WinUI, we need to create a style that has xaml content instead of just a ContentPresenter (because there's no native view to put in that ContentPresenter).

If updates are required in the package to allow that, let's do them now.

Reply:
Understood, I'll fix it soon.

Reply:
https://github.com/nventive/ExtendedSplashScreen/pull/64